app/models/patient_changeset.rb

```diff
@@ -135,6 +146,10 @@ def parent_relationships
         end
   end
 
+  def assign_patient_id!
```

Comment:
```suggestion
  def assign_patient_id
```

Just a small thing, but I would expect a method with `!` to save the record.